### PR TITLE
Added LicenseService APIs

### PR DESCRIPTION
**Pull request by sunithasomanna**
Added LicenseService APIs in the API readme

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -11655,6 +11656,141 @@ curl -i -X PATCH \
...
+| **Returns**        | Properties for viewing the service and links to the actual collections of manager licenses. |
+| **Response Code**  | `200 OK`                                                     |
+| **Authentication** | Yes                                                          |
+
+>**curl command**
+
+```
+curl -i GET \
+   -H "X-Auth-Token:{X-Auth-Token}" \
+ 'https://{odimra_host}:{port}/redfish/v1/LinceseService'
```
Comment (amar-shalgar) [inline docs/README.md]: Please correct the spelling for LinceseService

File: docs/README.md [hunk 2]
```diff
@@ -11655,6 +11656,141 @@ curl -i -X PATCH \
...
+
+>**Sample response body**
+
+```
+{
+   "@odata.context":"/redfish/v1/$metadata#LicenseService.LicenseService",
+   "@odata.id":"/redfish/v1/LicenseService",
+   "@odata.type":"#LicenseService.v1_0_0.LicenseService",
+   "Description":"License Service",
+   "Id":"",
```
Comment (amar-shalgar) [inline docs/README.md]: ID cannot be empty

File: docs/README.md [hunk 3]
```diff
@@ -11655,6 +11656,141 @@ curl -i -X PATCH \
...
+| **Returns**        | Links to the licenses collection.                            |
+| **Response Code**  | `200 OK`                                                     |
+| **Authentication** | Yes                                                          |
+
+>**curl command**
+
+```
+curl -i GET \
+   -H "X-Auth-Token:{X-Auth-Token}" \
+ 'https://{odimra_host}:{port}/redfish/v1/LinceseService/Licenses/'
```
Comment (amar-shalgar) [inline docs/README.md]: Please check the spelling

File: docs/README.md [hunk 4]
```diff
@@ -11655,6 +11656,141 @@ curl -i -X PATCH \
...
+
+```
+{
+   "@odata.context":"/redfish/v1/$metadata#License.License",
+   "@odata.id":"/redfish/v1/LicenseService/Licenses/8dd3fb4d-0429-4262-989f-906df092aefd.1.1",
+   "@odata.type":"#License.v1_0_0.License",
+   "Id":"8dd3fb4d-0429-4262-989f-906df092aefd.1.1",
+   "Name":"iLO License",
+   "Description":"License",
+   "LicenseType":"Perpetual"
```
Comment (amar-shalgar) [inline docs/README.md]: This response looks incomplete

**Review by amar-shalgar — CHANGES_REQUESTED** [review]
Please address changes

File: docs/README.md (hunk 1 above)
Comment (sunithasomanna) [reply docs/README.md]: done

File: docs/README.md (hunk 2 above)
Comment (sunithasomanna) [reply docs/README.md]: done

**Review by hemkapoor — APPROVED** [review]
LGTM

**Review by jeevan-kamkar — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**